src/readbin/compare.ml

```diff
@@ -0,0 +1,133 @@
+open Core_kernel.Std
+open Cmdliner
+open Bap.Std
+open Bap_plugins.Std
+
+exception No_unstripped_file
+
+let print_source = function
+  | `bw -> print_endline "bw"
+  | _ -> print_endline "others"
+
+let source = [
+  "BW", `bw;
+  "SymTbl", `symtbl;
+  "User", `user;
+  "Ida", `ida;
+]
+
+let source_gt = [
+  "User", `user;
+  "SymTbl", `symtbl
+]
+
+let tool : _ Term.t =
+  let doc = "The source of the function start result that we are going to
+  evaluate. " ^ (Arg.doc_alts_enum source) in
```

Comment:
Did you tried to read what you have wrote? I mean run with `--help`? Is it readable? Put yourself on the place of a user, that doesn't necessary know, what your program is doing. What is "The source"? is it the source code? What is "function start result"? Maybe it is better to say, "A name of tool to evaluate. Should be %s"